packages/server/__snapshots__/7_record_spec.js

```diff
@@ -26,7 +26,45 @@ Oops...we found an error preparing this test file:
 
 The error was:
 
-Error: Cannot find module '../it/does/not/exist' from '/foo/bar/.projects/e2e/cypress/integration'
+Error: Webpack Compilation Error
+./cypress/integration/record_error_spec.coffee
+Module not found: Error: Can't resolve '../it/does/not/exist' in '/foo/bar/.projects/e2e/cypress/integration'
+resolve '../it/does/not/exist' in '/foo/bar/.projects/e2e/cypress/integration'
+  using description file: /foo/bar/.projects/e2e/package.json (relative path: ./cypress/integration)
+    Field 'browser' doesn't contain a valid alias configuration
+    using description file: /foo/bar/.projects/e2e/package.json (relative path: ./cypress/it/does/not/exist)
+      no extension
+        Field 'browser' doesn't contain a valid alias configuration
+        /foo/bar/.projects/e2e/cypress/it/does/not/exist doesn't exist
+      .js
+        Field 'browser' doesn't contain a valid alias configuration
+        /foo/bar/.projects/e2e/cypress/it/does/not/exist.js doesn't exist
+      .json
+        Field 'browser' doesn't contain a valid alias configuration
+        /foo/bar/.projects/e2e/cypress/it/does/not/exist.json doesn't exist
+      .jsx
+        Field 'browser' doesn't contain a valid alias configuration
+        /foo/bar/.projects/e2e/cypress/it/does/not/exist.jsx doesn't exist
+      .coffee
+        Field 'browser' doesn't contain a valid alias configuration
+        /foo/bar/.projects/e2e/cypress/it/does/not/exist.coffee doesn't exist
+      .ts
+        Field 'browser' doesn't contain a valid alias configuration
+        /foo/bar/.projects/e2e/cypress/it/does/not/exist.ts doesn't exist
+      .tsx
+        Field 'browser' doesn't contain a valid alias configuration
+        /foo/bar/.projects/e2e/cypress/it/does/not/exist.tsx doesn't exist
+      as directory
+        /foo/bar/.projects/e2e/cypress/it/does/not/exist doesn't exist
+[/foo/bar/.projects/e2e/cypress/it/does/not/exist]
+[/foo/bar/.projects/e2e/cypress/it/does/not/exist.js]
+[/foo/bar/.projects/e2e/cypress/it/does/not/exist.json]
+[/foo/bar/.projects/e2e/cypress/it/does/not/exist.jsx]
+[/foo/bar/.projects/e2e/cypress/it/does/not/exist.coffee]
```

Comment:
do you know why all of these extra error messages are printed? I think we should try to prevent this from happening to users when only need the first line of the error message

Reply:
but maybe that isn't related to this PR and should instead be addressed in the webpack-preproc repo

Reply:
It's webpack being more verbose. It's listing all the extensions and paths where it looked for the file. I kind of like that aspect of it, though I agree it could be less verbose and still express the same thing.

It should only happen in this case, where the file doesn't exist. What do you think about editing it down so it looks like the following?

```
Error: Webpack Compilation Error
./cypress/integration/record_error_spec.coffee
Module not found: Error: Can't resolve '../it/does/not/exist' in '/foo/bar/.projects/e2e/cypress/integration'
Looked for and couldn't find the file at the following paths:
[/foo/bar/.projects/e2e/cypress/it/does/not/exist]
[/foo/bar/.projects/e2e/cypress/it/does/not/exist.js]
[/foo/bar/.projects/e2e/cypress/it/does/not/exist.json]
[/foo/bar/.projects/e2e/cypress/it/does/not/exist.jsx]
[/foo/bar/.projects/e2e/cypress/it/does/not/exist.coffee]
```

Reply:
@Bkucera Addressed here: https://github.com/cypress-io/cypress-webpack-preprocessor/pull/102. Once that's merged and released, I can update the dep and the snapshot here.

Reply:
Yes. I definitely like it being more explicit - with the shortened version